### PR TITLE
Fixed Crash When Pasting Code from iOS OneTimeCode

### PR DESCRIPTION
**Pull request by iTatla1**
There was bug when pasting code from clipboard i.e removing last character when textfield text count was zero.

### CODE REVIEW

**Comment by iTatla1** [comment]
One Time Code setup for digit input view

File: DigitInputView/Classes/DigitInputView.swift [hunk 1]
```diff
@@ -236,6 +248,19 @@ open class DigitInputView: UIView {
...
+        // .oneTimeCode content type available on iOS 12 and above devices
+        // One time code
+       
+        if isOneTimeCode {
+            if #available(iOS 12.0, *) {
+                textField?.textContentType = .oneTimeCode
+            }
+        }
+        else{
+            textField?.textContentType = nil
```
Comment (boranyldrm) [inline DigitInputView/Classes/DigitInputView.swift:261]: error on line 261 : _'textContentType' is only available on iOS 10.0 or newer_

Comment (boranyldrm) [reply DigitInputView/Classes/DigitInputView.swift:261]:
```suggestion
            if #available(iOS 10.0, *) {
                textField?.textContentType = nil
            }
```

**Comment by boranyldrm** [comment]
This Pull Request solves the issue #9

**Comment by mnvoh** [comment]
@iTatla1 @boranyldrm Thanks for the contribution guys

**Pull request merged**